ci/build_python.sh

```diff
@@ -7,7 +7,7 @@ rapids-print-env
 CMAKE_GENERATOR=Ninja \
 CONDA_OVERRIDE_CUDA="${RAPIDS_CUDA_VERSION}" \
 rapids-conda-retry mambabuild \
-    --channel legate \
+    --channel legate/label/experimental \
```

Comment:
```suggestion
    --channel legate \
    --channel legate/label/experimental \
```

Both channels are needed here. Eventually there will be e.g. a `cunumeric=24.09.00` on the `main` label at https://anaconda.org/legate/cunumeric/files, and without `--channel legate` and at that point we'd want the build here to pick that up instead of `cunumeric` / `legate` nightlies.